### PR TITLE
Fix empty catch in error handling

### PR DESCRIPTION
**Pull request by akiratk0355**
Connected to #33

### CODE REVIEW

File: App/src/main/java/cc/softwarefactory/lokki/android/MainApplication.java [hunk 1]
```diff
@@ -62,7 +62,9 @@ protected int sizeOf(String key, Bitmap bitmap) {
         if (!iDontWantToSeeString.isEmpty()) {
             try {
                 MainApplication.iDontWantToSee = new JSONObject(iDontWantToSeeString);
-            } catch (JSONException ex) {
+            } catch (JSONException e) {
+                MainApplication.iDontWantToSee = null;
```
Comment (Kauhsa) [inline App/src/main/java/cc/softwarefactory/lokki/android/MainApplication.java]: If I follow the code correctly, in this case it's going to be null anyway – but yeah it's not a bad idea to be explicit.

**Comment by Kauhsa** [comment]
Looks good to me. Does this fix all of the empty catch blocks?

File: App/src/main/java/cc/softwarefactory/lokki/android/MainActivity.java [hunk 2]
```diff
@@ -458,6 +458,7 @@ public void onClick(DialogInterface dialog, int which) {
             try {
                 alertDialog.show();
             } catch (Exception ex) {
```
Comment (eemihauk) [inline App/src/main/java/cc/softwarefactory/lokki/android/MainActivity.java]: The `alertDialog.show()` doesn't actually require a try-catch block so they can be removed.

**Comment by akiratk0355** [comment]
There seems to be nothing else according to lint check

**Comment by eemihauk** [comment]
@Kauhsa I have been pedantic that when someone is refactoring catch blocks, the catch blocks should catch specific exceptions rather than all exceptions as in our [guidelines](http://source.android.com/source/code-style.html#dont-catch-generic-exception).

**Comment by Kauhsa** [comment]
@eemihauk which is, of course, correct – I just wasn't careful enough ;)

**Pull request merged**